mne/utils.py

```diff
@@ -2388,8 +2388,9 @@ def _check_ch_locs(chs):
         The channels from info['chs']
     """
     locs3d = np.array([ch['loc'][:3] for ch in chs])
-    return (locs3d == 0).all() or \
-        (~np.isfinite(locs3d)).all() or np.allclose(locs3d, 0.)
+    return not ((locs3d == 0).all() or
```

Comment:
why the `not` ?

Reply:
Because it's counterintuitive for it to return True if there are no channel locations.

Reply:
okay fair enough. I'd recommend adding a test to catch the corner case you found ...

Reply:
What do you mean by corner case?

Reply:
I mean you probably need a test so that if someone edits around L2131, they will get a failing test. Although since it's a viz function, maybe it's not so straightforward ...